### PR TITLE
Hackathon23

### PR DESCRIPTION
**Pull request by GerardoAcquaviva**
I added the Prettier and subworkflow sections, task is yet to be complemented by @Furentsu's work.

<a href="https://gitpod.io/#https://github.com/nf-core/nf-co.re/pull/1705"></a>

### CODE REVIEW

File: markdown/developers/tutorials/nf_core_contributing_overview.md [hunk 1]
```diff
@@ -255,6 +255,25 @@ It will check:
...
+```bash
+prettier -c .
+```
+
+While the command to automatically format the style of all the files in your directory with Prettier is the following:
+
+```bash
+prettier -w .
+```
+
```
Comment (mashehu) [inline markdown/developers/tutorials/nf_core_contributing_overview.md:276]:
```suggestion
// TODO: mention pre-commit here after we add it to the template.
```

File: markdown/developers/tutorials/nf_core_contributing_overview.md [hunk 2]
```diff
@@ -334,6 +353,11 @@ A step-by-step tutorial [adding new modules](https://nf-co.re/docs/contributing/
 - Use the pipeline you created in Exercise 2 and add an nf-core module (e.g. trimgalore).
 - Connect the module to the main pipeline workflow (under `workflows/<pipeline-name>.nf`).
 
+#### Subworkflow
+
+Subworkflows are chains of modules that offer a higher-level of functionality within the context of a pipeline. A subworkflow is able tu run multiple tools on the same input file.
```
Comment (mashehu) [inline markdown/developers/tutorials/nf_core_contributing_overview.md:358]:
```suggestion
Subworkflows are chains of modules that offer a higher-level of functionality within the context of a pipeline. A subworkflow is able to run multiple tools on the same input file.
```

**Review by mashehu — CHANGES_REQUESTED** [review]

File: markdown/developers/tutorials/nf_core_contributing_overview.md (hunk 1 above)
Comment (GerardoAcquaviva) [reply markdown/developers/tutorials/nf_core_contributing_overview.md:276]: can you be more precise? do you want me to add another paragraph explaining pre-commit?

Comment (maxulysse) [reply markdown/developers/tutorials/nf_core_contributing_overview.md:277]: @mashehu what did you meant here?

**Review by mashehu — APPROVED** [review]

File: markdown/developers/tutorials/nf_core_contributing_overview.md (hunk 1 above)
Comment (mashehu) [reply markdown/developers/tutorials/nf_core_contributing_overview.md:277]: Is resolved now

File: package-lock.json [hunk 3]
```diff
@@ -12,9 +12,6 @@
                 "@highlightjs/cdn-assets": "^11.6.0",
                 "autoprefixer": "^10.0.2",
                 "aws-sdk": "^2.994.0",
```
Comment (maxulysse) [inline package-lock.json:14]:
```suggestion
                "aws-sdk": "^2.994.0",
                "bootstrap": "^5.1.1",
                "bootstrap-dark-5": "1.1.1",
                "bootstrap-print-css": "^1.0.1",
```

File: package-lock.json [hunk 4]
```diff
@@ -33,6 +30,9 @@
             },
             "devDependencies": {
                 "@prettier/plugin-php": "^0.18.4",
+                "bootstrap": "5.1.1",
+                "bootstrap-dark-5": "1.1.1",
+                "bootstrap-print-css": "^1.0.1",
```
Comment (maxulysse) [inline package-lock.json:35]:
```suggestion
```

File: package-lock.json [hunk 5]
```diff
@@ -80,6 +80,7 @@
             "version": "2.11.5",
             "resolved": "https://registry.npmjs.org/@popperjs/core/-/core-2.11.5.tgz",
             "integrity": "sha512-9X2obfABZuDVLCgPK9aX0a/x4jaOEweTTWE2+9sr0Qqqevj2Uv5XorvusThmc9XGYpS9yI+fhh8RTafBtGposw==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:83]:
```suggestion
```

File: package-lock.json [hunk 6]
```diff
@@ -239,6 +240,7 @@
             "version": "5.1.1",
             "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-5.1.1.tgz",
             "integrity": "sha512-/jUa4sSuDZWlDLQ1gwQQR8uoYSvLJzDd8m5o6bPKh3asLAMYVZKdRCjb1joUd5WXf0WwCNzd2EjwQQhupou0dA==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:243]:
```suggestion
```

File: package-lock.json [hunk 7]
```diff
@@ -251,6 +253,7 @@
             "version": "1.1.1",
             "resolved": "https://registry.npmjs.org/bootstrap-dark-5/-/bootstrap-dark-5-1.1.1.tgz",
             "integrity": "sha512-P7gRjU6Iog/ULKvYbV7u99Ix012occPV2bOjh3z+8CwraixS4UL71S247e1/Mm9fgDIuN/PscGXLC8GS2ZqU4g==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:256]:
```suggestion
```

File: package-lock.json [hunk 8]
```diff
@@ -261,7 +264,8 @@
         "node_modules/bootstrap-print-css": {
             "version": "1.0.1",
             "resolved": "https://registry.npmjs.org/bootstrap-print-css/-/bootstrap-print-css-1.0.1.tgz",
-            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w=="
+            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w==",
+            "dev": true
```
Comment (maxulysse) [inline package-lock.json:268]:
```suggestion
            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w=="
```

File: package-lock.json [hunk 9]
```diff
@@ -1626,6 +1630,7 @@
             "version": "2.11.5",
             "resolved": "https://registry.npmjs.org/@popperjs/core/-/core-2.11.5.tgz",
             "integrity": "sha512-9X2obfABZuDVLCgPK9aX0a/x4jaOEweTTWE2+9sr0Qqqevj2Uv5XorvusThmc9XGYpS9yI+fhh8RTafBtGposw==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:1633]:
```suggestion
```

File: package-lock.json [hunk 10]
```diff
@@ -1731,20 +1736,23 @@
             "version": "5.1.1",
             "resolved": "https://registry.npmjs.org/bootstrap/-/bootstrap-5.1.1.tgz",
             "integrity": "sha512-/jUa4sSuDZWlDLQ1gwQQR8uoYSvLJzDd8m5o6bPKh3asLAMYVZKdRCjb1joUd5WXf0WwCNzd2EjwQQhupou0dA==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:1739]:
```suggestion
```

File: package-lock.json [hunk 11]
```diff
@@ -1731,20 +1736,23 @@
...
             "requires": {}
         },
         "bootstrap-dark-5": {
             "version": "1.1.1",
             "resolved": "https://registry.npmjs.org/bootstrap-dark-5/-/bootstrap-dark-5-1.1.1.tgz",
             "integrity": "sha512-P7gRjU6Iog/ULKvYbV7u99Ix012occPV2bOjh3z+8CwraixS4UL71S247e1/Mm9fgDIuN/PscGXLC8GS2ZqU4g==",
+            "dev": true,
```
Comment (maxulysse) [inline package-lock.json:1746]:
```suggestion
```

File: package-lock.json [hunk 12]
```diff
@@ -1731,20 +1736,23 @@
...
             "requires": {
                 "bootstrap": "^5.1.1"
             }
         },
         "bootstrap-print-css": {
             "version": "1.0.1",
             "resolved": "https://registry.npmjs.org/bootstrap-print-css/-/bootstrap-print-css-1.0.1.tgz",
-            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w=="
+            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w==",
+            "dev": true
```
Comment (maxulysse) [inline package-lock.json:1755]:
```suggestion
            "integrity": "sha512-I73Cw87BaxCccTjo3qEbvn7KRb55msMxTuT7mpkAAY4Obq8iG9xCybdxnJqq+RrykLD79O3092AiJwaKiEex7w=="
```

**Review by maxulysse — APPROVED** [review]

**Comment by maxulysse** [comment]
@nf-core-bot fix linting pretty please :pray:

**Pull request merged**